AppCenter/AppCenter/Internals/Ingestion/MSACIngestionProtocol.h

```diff
@@ -20,7 +20,7 @@ typedef void (^MSACSendAsyncCompletionHandler)(NSString *callId, NSHTTPURLRespon
 /**
  * The indicator of readiness to send data.
  */
-@property(nonatomic, readonly, getter=isReadyToSend) BOOL readyToSend;
+@property(nonatomic, readonly, getter=isReadyToSend) BOOL readyToSend NS_SWIFT_NAME(readyToSend);
```

Comment:
Probably we can take away ` NS_SWIFT_NAME(readyToSend)`, as the property already has the same name.